core/src/main/java/com/statful/client/core/StatfulClientImpl.java

```diff
@@ -93,6 +93,17 @@ public final SenderFacade gauge(final String metricName, final Integer value) {
     }
 
     @Override
+    public final SenderFacade metric(final String metricName, final Integer value) {
+        SenderAPI senderAPI = MetricsSenderAPI.newInstance(this).with()
+                .configuration(configuration)
+                .aggFreq(configuration.getDefaultAggregationFreq())
```

Comment:
No, this is the construct to create the `put` metric in the client interface using the builder pattern. This basically returns a builder for `put` with preset values. If you check above, the code preset all defaults for timer, gauge and counter in the same way.